angular-primeng-app/src/app/components/post-details/post-details.component.html

```diff
@@ -5,9 +5,18 @@ <h1 class="title">{{ post.title }}</h1>
       <img class="cover-image" [src]="post.coverImage.url" alt="Cover image for {{ post.title }}">
       <div class="post-details">
         <div class="author-info">
-          <img class="author-image" [src]="post.author.profilePicture" alt="{{ post.author.name}}">
+          @if (isTeam) {
+            <p-avatarGroup styleClass="mb-3" >
+               <p-avatar [image]="post.author.profilePicture" size="large" shape="circle" title="{{ post.author.name}}"></p-avatar>
+               @for (coAuthor of post.coAuthors; track coAuthor.username) {
+                 <p-avatar [image]="coAuthor.profilePicture" size="large"  shape="circle" title="{{ coAuthor.username}}"></p-avatar>
+                }
+            </p-avatarGroup>
+          } @else {
+             <img class="author-image" [src]="post.author.profilePicture" alt="{{ post.author.name}}">
```

Comment:
@DongreJaipal looks great one tiny thing - here in the @else statement - let's remain consistent and use 

```
<p-avatar [image]="post.author.profilePicture" size="large" shape="circle" title="{{ post.author.name}}"></p-avatar>
```